explainaboard/tests/artifacts/machine_translation/output_with_features.json

```diff
@@ -1,11 +1,21 @@
 {
   "metadata": {
     "custom_features": {
-      "num_capital_letters": {
-        "dtype": "float",
-        "description": "number of capital letters in the string",
-        "num_buckets": 4
+      "example": {
+        "num_capital_letters": {
+          "dtype": "float"
```

Comment:
we probably also want to provide an argument to take in customized feature functions in the future.

Reply:
I agree, there's an issue open for this: https://github.com/neulab/ExplainaBoard/issues/338